src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -371,6 +371,9 @@ void GNAPlugin::InitGNADevice() {
                     GetDeviceVersionFromString(config.gnaCompileTarget));
         size_t page_size_bytes = 4096;
         gnamem = std::make_shared<gna_memory_device>(memory::GNAAllocator(gnadevice), page_size_bytes);
+        if (gnaFlags->log_level == ov::log::Level::DEBUG) {
+            gnadevice->debugLogEnabled = true;
```

Comment:
Maybe using gnadevice->log_level would be a more flexible approach. Besides gnadevice could be changed to either gna_device or gnaDevice to align the code style.